schema/SystemStructureCommon.xsd

```diff
@@ -274,15 +274,15 @@
                                 </xs:documentation>
                             </xs:annotation>
                         </xs:element>
-                        <xs:element name="Signature" type="ssc:SignatureType" minOccurs="0" maxOccurs="unbounded">
+                        <xs:group ref="ssc:GSignature">
                             <xs:annotation>
                                 <xs:documentation xml:lang="en">
                                     This element can contain digital signature information on the meta data referenced
```

Comment:
Minor issue:
This annotation at the instance of the GSignature group is a duplicate of the comment at the group definition (line 349).
If I look at the SSP schema in XML Spy, I see sometimes groups with annotations at the usages of the group, and sometimes not.
![image](https://github.com/user-attachments/assets/117d7fbc-1ffb-41c2-82af-b91a65757872)
I think, in this case, the Annotation can be deleted here.

Reply:
I retained the annotation as it is not a full duplicate of the definition annotation, as this puts the relationship to the meta data into the text (and not just any enclosing data), so might be slightly clearer. We might want to revisit our use of annotations going forward in a more consistent style, however keeping it like this for now.